float32 lat
float32 lon
float32 heading
float32 velocity
float32 angZ
